docs/content/preview/yugabyte-cloud/cloud-monitor/cloud-alerts.md

```diff
@@ -127,14 +124,44 @@ If your cluster generates this alert, you may need to rate limit your queries. L
 
 If your cluster generates this alert but isn't under a very large workload, contact {{% support-cloud %}}.
 
+#### Fix memory alerts
+
+YugabyteDB Managed sends a notification when memory use in the cluster exceeds the threshold, as follows:
+
+- Memory use exceeds 75% on average for at least 10 minutes (Warning).
+- Memory use exceeds 90% on average for at least 10 minutes (Severe).
+
+If your cluster experiences frequent spikes in memory use, consider optimizing your workload.
+
+Unoptimized queries can lead to memory alerts. Use the [Slow Queries](../cloud-queries-slow/) and [Live Queries](../cloud-queries-live/) views to identify potentially problematic queries, then use the EXPLAIN statement to see the query execution plan and identify optimizations. Consider adding one or more indexes to improve query performance. For more information, refer to [Analyzing Queries with EXPLAIN](../../../explore/query-1-performance/explain-analyze/).
+
+High memory use could also indicate a problem and may require debugging by {{% support-cloud %}}.
+
+If memory use is continuously higher than 80%, your workload may also exceed the capacity of your cluster. Consider scaling your cluster by adding vCPUs. Refer to [Scale and configure clusters](../../cloud-clusters/configure-clusters/).
+
+#### Fix YSQL connection alerts
+
+YugabyteDB Managed sends a notification when the number of YSQL connections on any node in the cluster exceeds the threshold, as follows:
+
+- YSQL connections exceeds 80% of the limit (Warning).
+- YSQL connections exceeds 95% of the limit (Severe).
```

Comment:
I just changed these to 60% and 95% respectively today. I have a PR out that will fix the UI text to show the right values in the alert configuration.